source/user-manual/reference/tools/wazuh-control.rst

```diff
@@ -26,6 +28,10 @@ The wazuh-control script is used to start, stop, configure, check on the status
 |             | This option is not available on a local Wazuh installation.                                             |
 +-------------+---------------------------------------------------------------------------------------------------------+
 | **status**  | Determine which Wazuh processes are running.                                                            |
++-------------+---------------------------------------------------------------------------------------------------------+
+| **info**    | Prints the installation type, beside the Wazuh Version and Revision, in NAME="value" pairs              |
```

Comment:
Let's be consistent with the explanation of all the options. So, put a `.` (dot) at the end of the explanations.

Reply:
Done!